src/app.py

```diff
@@ -82,26 +84,37 @@ def __init__(self, parent=None):
         loadDatabaseButton.clicked.connect(self.loadDatabase)
         self.leftPaneView.addWidget(loadDatabaseButton)
 
+        self.dropDown = QComboBox()
+        self.dropDown.addItems(self.getCleanedTestIds())
+        self.leftPaneView.addWidget(self.dropDown)
+
         self.leftPaneView.addWidget(self.markerListPlaceholder)
         self.rightPaneView.addWidget(self.currentDisplayingGraph)
 
         mainWindowView.addWidget(leftPane)
         mainWindowView.addWidget(rightPane)
 
 
+    def getCleanedTestIds(self):
```

Comment:
Sensible implementation - hopefully my slightly messy code didn't give you much trouble!